torch/_dynamo/variables/functions.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, fn, is_constant=False, **kwargs):
             self.is_constant = False
 
         assert isinstance(
-            fn, (types.FunctionType, torch.jit.ScriptFunction)
+            fn, (types.FunctionType, torch.jit.ScriptFunction, functools.partial)
```

Comment:
functools.partial are different from functions, and in all likelihood will need to be modeled by its own VariableTracker subclass. See https://docs.python.org/3/library/functools.html#partial-objects

Reply:
I seem to get along fine modeling them as Function VTs that do a little extra at call_function time...

Reply:
I'm just saying it's inaccurate and if someone eventually needs to make, e.g., type() or partial.func to work, they will have to undo what you did here